src/dotnet/UI.Blazor/Components/SearchPanel/search-panel.ts

```diff
@@ -0,0 +1,50 @@
+import { Subject, } from 'rxjs';
+
+import { Log } from 'logging';
+import { setTimeout } from 'timerQueue';
+
+const { debugLog } = Log.get('SearchPanel');
+
+enum Side {
+    Left,
+    Right,
+}
+
+export class SearchPanel {
```

Comment:
Ok, this is not going to work. You just designed an extra panel instead of adding search mode to the right panel. IMO this is wrong - it won't be swipeable, etc.

Reply:
I.e. I'd rather add a search mode to the right panel.